.github/workflows/test-suite.yml

```diff
@@ -61,14 +65,15 @@ jobs:
     - name: Install Foundry (anvil)
       uses: foundry-rs/foundry-toolchain@v1
     - name: Run tests in release
-      run: make test-release
+      run: WATCH_HOST=host.docker.internal make test-release
```

Comment:
I _think_ this may not work on GitHub hosted runners if they're not running in docker containers, perhaps we can leave it empty if `env.SELF_HOSTED_RUNNERS == false` so it defaults to `localhost`?

Reply:
Good call, I had a shot at fixing this in https://github.com/sigp/lighthouse/pull/4506/commits/ce4ed6e1e9cefa6156bfdef3abd9c1040e2d5f08.